test/collection.js

```diff
@@ -628,6 +628,18 @@ describe('Chained Collection', () => {
     ).to.equal([6, 10, 20, 30])
   })
 
+  it('pop', async () => {
+    expect(
+      await Collect([1, 2, 3])
+        .pop()
+    ).to.equal(3)
+
+    expect(
+      await Collect([])
+        .pop()
+    ).to.equal(undefined)
+  })
```

Comment:
Can you please add a test that checks if the original array is changed (the last item removed)?

Reply:
Sure